From 9860f3ad1c5d76e8c5100454b8ce6825bc293403 Mon Sep 17 00:00:00 2001
From: Jeff Vander Stoep <jeffv@google.com>
Date: Fri, 20 Oct 2017 13:35:42 -0700
Subject: [PATCH 5/8] priv_app: move logspam suppression to core policy

No sign of these denials getting cleaned up, so supress them in core
policy.

Test: build
Change-Id: I0320425cb72cbd15cef0762090899491338d4f7c
---
 private/priv_app.te | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/private/priv_app.te b/private/priv_app.te
index 60fb4110..f4cfc173 100644
--- a/private/priv_app.te
+++ b/private/priv_app.te
@@ -114,6 +114,9 @@ read_runtime_log_tags(priv_app)
 
 # suppress denials when safetynet scans /system
 dontaudit priv_app exec_type:file getattr;
+dontaudit priv_app device:dir read;
+dontaudit priv_app proc_interrupts:file read;
+dontaudit priv_app proc_modules:file read;
 
 ###
 ### neverallow rules
-- 
2.17.0

